retesteth/ethObjects/object.cpp

```diff
@@ -237,52 +237,39 @@ DataObject object::prepareGenesisParams(std::string const& _network, std::string
 {
     ClientConfig const& cfg = Options::get().getDynamicOptions().getCurrentConfig();
     test::checkAllowedNetwork(_network, cfg.getNetworks());
-    bool networkChecked = false;
```

Comment:
this function I want to move to .json configs. it looks way to hardcode. 
the chain params config might be put in a .json in ini folder.